### PR TITLE
Home page: cpu hours key error

### PR DESCRIPTION
**Pull request by kate-v-stepanova**

### CODE REVIEW

File: status/util.py [hunk 1]
```diff
@@ -133,14 +133,16 @@ def get(self):
         instruments = self.application.server_status['instruments']
         server_status = {}
         for row in reduced_rows:
-            server = row.value['name']
+            server = row.value.get('name')
+            if server is None:
```
Comment (Galithil) [inline status/util.py]: This can also be written as `if not server`, but I don't mind either way.

**Comment by kate-v-stepanova** [comment]
I tried to replace all the ['cpu hours'] with .get('cpu hours'), and all the other [] as well. Seems like I have problems with logic, because every time I fix this page, something else happens and it breaks the whole thing again. Sorry, I really hope it won't cause problems anymore :(

File: status/util.py [hunk 2]
```diff
@@ -158,7 +160,8 @@ def get(self):
             if not uppmax_ids:
                 break
             project_id = row.value['project'].replace('/', '_')
-            project_nobackup = copy.copy(row.value['project'].split('/')[0])
+            project_id = row.value.get('project', '').replace('/', '_')
+            project_nobackup = copy.copy(project_id.split('/')[0])
```
Comment (Galithil) [inline status/util.py]: L162 should go away.
L164 used to split on `/` but now it has been replaced with `_` , so I don't think it will work.

**Comment by Galithil** [comment]
L163 is still not going to work as you replace the `/` with `_` before trying to split on them.

**Comment by kate-v-stepanova** [comment]
don't close it now, I'm checking one more thing

**Comment by kate-v-stepanova** [comment]
Just one more thing before closing it, I'm checking it

**Comment by kate-v-stepanova** [comment]
any more comments, or should we merge this one? @Galithil, @ewels ?

File: status/util.py (hunk 1 above)
Comment (ewels) [inline status/util.py]: _* adds to python knowledge *_

**Pull request merged**